data-source/jsonnet/england-wales/household/blocks/who-lives-here/anyone_else_temporarily_away_list_collector.jsonnet

```diff
@@ -187,6 +187,11 @@ local editQuestion(questionTitle) = {
     question: {
       id: 'anyone-else-temp-away-remove-question',
       type: 'General',
+      guidance: {
+        contents: [{
+          title: 'All of the data entered about this person will be deleted',
```

Comment:
Need to verify whether this is to be displayed on the other list collectors also?. As this will only be displayed for removing people from anyone else temp away list collector.

Reply:
Yeah I was going to clarify that with Donna but forgot, the card only mentioned that one though

Reply:
Donna has confirmed that these should be on all so I have made the change